.circleci/config.yml

```diff
@@ -41,13 +62,27 @@ jobs:
       - deploy:
           name: create GitHub release
           command: |
-            export PREV_RELEASE=$(git tag --sort=version:refname | tail -n2 | head -n1)
```

Comment:
I'm not a fan of the duplication here, but if we're releasing the projects separately then I'd rather spend my time doing that instead of fixing this code :)